include/libdnf/base/transaction.hpp

```diff
@@ -23,6 +23,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include "libdnf/base/base_weak.hpp"
 #include "libdnf/base/goal_elements.hpp"
+#include "libdnf/base/log_event.hpp"
```

Comment:
You probably forgot to add this header file to the PR.

Reply:
Thanks, fixed